processing/src/test/java/io/druid/query/aggregation/hyperloglog/HyperUniquesAggregationTest.java

```diff
@@ -0,0 +1,99 @@
+/*
+* Licensed to Metamarkets Group Inc. (Metamarkets) under one
+* or more contributor license agreements. See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership. Metamarkets licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License. You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+package io.druid.query.aggregation.hyperloglog;
+
+import com.google.common.collect.Lists;
+import com.google.common.io.Files;
+import com.metamx.common.guava.Sequence;
+import com.metamx.common.guava.Sequences;
+import io.druid.data.input.MapBasedRow;
+import io.druid.granularity.QueryGranularity;
+import io.druid.jackson.AggregatorsModule;
+import io.druid.query.aggregation.AggregationTestHelper;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.File;
+
+public class HyperUniquesAggregationTest
+{
+  @Test
+  public void testIngestAndQuery() throws Exception
+  {
+    AggregationTestHelper helper = new AggregationTestHelper(Lists.newArrayList(new AggregatorsModule()));
+
+    String metricSpec = "[{"
+                        + "\"type\": \"hyperUnique\","
+                        + "\"name\": \"index_hll\","
+                        + "\"fieldName\": \"market\""
+                        + "}]";
+
+    String parseSpec = "{"
```

Comment:
we can probably move the parseSpec and the input data into the aggregation test helper?